src/hal/src/lib.rs

```diff
@@ -306,7 +306,7 @@ pub trait Instance {
 }
 
 /// The `Backend` trait wraps together all the types needed
-/// to implement a graphics backend.  Each backend, such as OpenGL
+/// for a graphics backend.  Each backend module, such as OpenGL
```

Comment:
double space found

Reply:
Unfortunately I was taught that you always put two spaces between sentences.  Markdown and HTML don't care but it's a hard habit to break.  I'll try to keep it consistent with your style.